--- gxml-0.10.0/gxml/gxml-init.vala.orig	2015-11-04 20:13:59.000000000 +0100
+++ gxml-0.10.0/gxml/gxml-init.vala	2016-09-23 21:49:12.100780960 +0200
@@ -1,4 +1,4 @@
-/**
+/*
  *  GXml
  *
  *  Authors:
@@ -6,21 +6,21 @@
  *       Daniel Espinosa <esodan@gmail.com>
  *
  *
- *  Copyright (c) 2015 Daniel Espinosa
- *  
- *  This library is free software; you can redistribute it and/or
- *  modify it under the terms of the GNU Lesser General Public
- *  License as published by the Free Software Foundation; either
- *  version 3 of the License, or (at your option) any later version.
- *  
- *  This library is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- *  Lesser General Public License for more details.
- *  
- *  You should have received a copy of the GNU Lesser General Public
- *  License along with this library; if not, write to the Free Software
- *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *  Copyright (c) 2015 Daniel Espinosa
+ *  
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 3 of the License, or (at your option) any later version.
+ *  
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *  
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 namespace GXml {
   class Init {
@@ -33,4 +33,4 @@
       }
     }
   }
-}
\ No newline at the end of file
+}
--- gxml-0.10.0/docs/valadoc/gtk-doc/Makefile.am.orig	2015-11-04 20:13:59.000000000 +0100
+++ gxml-0.10.0/docs/valadoc/gtk-doc/Makefile.am	2016-09-23 21:38:45.854121445 +0200
@@ -12,7 +12,8 @@
 	--pkg gee-0.8 \
 	--vapidir=$(top_srcdir)/gxml \
 	--vapidir=$(top_srcdir)/vapi \
-	$(top_srcdir)/vapi/xlibxml-1.0.vapi
+	$(top_srcdir)/vapi/xlibxml-1.0.vapi \
+	$(top_srcdir)/vapi/config.vapi
 
 sources= \
 	$(top_srcdir)/gxml/*.vala
--- gxml-0.10.0/docs/valadoc/gir-docs/Makefile.am.orig	2015-07-08 20:11:44.000000000 +0200
+++ gxml-0.10.0/docs/valadoc/gir-docs/Makefile.am	2016-09-23 21:51:31.140779742 +0200
@@ -12,7 +12,8 @@
 	--pkg gee-0.8 \
 	--vapidir=$(top_srcdir)/gxml \
 	--vapidir=$(top_srcdir)/vapi \
-	$(top_srcdir)/vapi/xlibxml-1.0.vapi
+	$(top_srcdir)/vapi/xlibxml-1.0.vapi \
+	$(top_srcdir)/vapi/config.vapi
 
 sources= \
 	$(top_srcdir)/gxml/*.vala
