--- gxml-0.10.0/gxml/libxml-NamedAttrMap.vala.orig	2016-02-22 20:23:53.000000000 +0100
+++ gxml-0.10.0/gxml/libxml-NamedAttrMap.vala	2016-09-23 20:55:54.587484135 +0200
@@ -32,8 +32,8 @@
 	 * {@link Gee.Iterable} interfaces.
 	 */
 	public abstract class AbstractNamedAttrMap : Object,
-		Traversable<Entry<string,GXml.Attribute>>,
-		Iterable<Entry<string,GXml.Attribute>>
+		Traversable<Entry>,
+		Iterable<Entry>
 	{
 		protected xElement elem;
 
